innovation.game.php

```diff
@@ -22829,7 +22869,16 @@ function stInterInteractionStep() {
                     self::executeDraw($player_id, $value_to_score, 'score');
                     self::executeDraw($player_id, $value_to_score, 'score');
                     break;     
-                
+                // id 413, Echoes age 8: Crossword
+                case "413N1A":
+                    if(array_sum(self::getAuxiliaryArray()) > 0) {
+                        $step--;
+                        self::incrementStep(-1);
+                    } else {
+                        $step++;
```

Comment:
What step are you skipping here?

Only the most complicated state machines within a card would require skipping a step.

Reply:
Maybe I am misunderstanding how $step works entirely; I was trying to keep all logic happening in "431N1A" steps, as the choice selection needs to be repeated N times until all N cards have been drawn (as the player controls the order). So, decrease the $step to keep things at 0, and increase step to finally end the dogma.

That said, I was iffy on the difference between $step-- and incrementStep(-1). Mostly, I was trying to copy the logic of other repeated choice cards I saw—Lever (which kahlia mentioned basing this off), Oars.